liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyLanguageServer.java

```diff
@@ -76,14 +78,20 @@ public synchronized void updateSettings(Object initializationOptionsSettings) {
         // TODO: else update settings
     }
 
+    @Override
```

Comment:
Are these really overrides? I don't see method connect, shutdown or exit in the class that is being extended. I see these were already overrides before, but I think that is incorrect. Also, is there a need to use `super` when the method is not in the current class and is inherited?

Reply:
Hmm, you are right, they're not overriding any superclass methods, but these methods are implementing the interface methods.
I think it is still a good marker to show it is an implemented method, even if it's not necessary. (I also read that Eclipse auto-marks implemented methods with `@Override` with Java 1.6+)
`super` is also not necessary, and serves more as an indicator. Should I remove these?

Reply:
k...if Eclipse auto-marks them that is fine. Just leave it.